src/utils/parser/tests/fixtures/var.js

```diff
@@ -1,5 +1,4 @@
 var foo = 1;
 let bar = 2;
 const baz = 3;
-const a = 4,
-  b = 5;
```

Comment:
this actually should be this way **becase** we're testing the weird behavior :) We want to make sure we find those vars too :P 

@ruturajv did this i believe